platform/linux-generic/odp_schedule_if.c

```diff
@@ -6,6 +6,12 @@
 
 #include <odp_schedule_if.h>
 
+extern const queue_api_t queue_default_api;
+extern const queue_fn_t queue_default_fn;
```

Comment:
This file should be renamed odp_component_if.c to match the earlier recommended rename of odp_schedule_if.h to odp_component_if.h.